ui/src/store/contract.ts

```diff
@@ -1,9 +1,8 @@
 import { create } from 'zustand';
 
 interface ContractState {
-  instance?: unknown;
+  instances?: Record<string, unknown>;
```

Comment:
oh... so it's _not_ orthogonal to this PR. elsewhere this is called `instance`; i.e. `agoricNames.instance.orca`. (likewise `brand`, `issuer`, ...)

OTOH, in the Zoe API, `getTerms()` returns something with `issuers` and `brands`... so... go figure.